src/main/java/org/pac4j/jax/rs/pac4j/JaxRsApplicationLogoutLogic.java

```diff
@@ -0,0 +1,16 @@
+package org.pac4j.jax.rs.pac4j;
+
+import org.pac4j.core.engine.DefaultApplicationLogoutLogic;
+
+/**
+ * @author Victor Noel - Linagora
+ * @since 1.1.1
+ * 
+ */
+public class JaxRsApplicationLogoutLogic<C extends JaxRsContext> extends DefaultApplicationLogoutLogic<Object, C> {
```

Comment:
At first I didn't even create those classes then I thought that if someone wanted to extend one of the logics, they might as well have a base class specific for JAX-RS.

It's good to me if instead we have as you propose:
 - logics using the profile manager configured in the config
 - filters that can set a default value for the profile manager (as in the discussion in pac4j/pac4j#803) if there is no profile manager set in the config
 - ways for users to override the both the config and the default one PER filter/logic!